### PR TITLE
New poster reminding captains to SECURE DA FECKIN DISK!!

### PR DESCRIPTION
**Pull request by etherware-novice**
## About The Pull Request
Adds a new poster for the captain as a reminder to secure the disk, and also adding the poster in caps quarters to be 100% sure the captain gets the message

## Why It's Good For The Game
Always good to have a reminder for the DISK

## Testing Photographs and Procedure

<details>
<summary>Screenshots&Videos</summary>

disk_onwall

</details>

## Changelog
:cl:
add: Nanotrasen has started a new initiative to remind captains about the disk, following a string of nuke op successes.
/:cl:

### CODE REVIEW

**Comment by PowerfulBacon** [comment]
Secure some bitches

**Comment by etherware-novice** [comment]
drip poster would also be a fun(ny) addition

**Comment by itsmeow** [comment]
drip is LRP

**Comment by Oscari-pog** [comment]
> drip is LRP

Swag shoes and Swag clothes are a thing though. So we need a Swag poster.

**Review by PowerfulBacon — CHANGES_REQUESTED** [review]
Walls should only have 1 item on them. You can't have a poster inside a light switch

**Comment by FortSumter** [comment]
Make the drip poster, but have it be a contraband for the floor bags and crate. Just because its unprofessional doesnt mean it can't exist in MRP.

Remember the crew are here to work, the players are here to have fun.

**Comment by etherware-novice** [comment]
> Walls should only have 1 item on them. You can't have a poster inside a light switch

Which map is this affected by?

**Comment by PowerfulBacon** [comment]
At least this one
image

**Comment by etherware-novice** [comment]
oh i dont remember which map was loaded...strongDMM diving time yey

File: code/game/objects/effects/contraband.dm [hunk 1]
```diff
@@ -657,6 +657,11 @@
 	desc = "This outdated informational poster uses Safety Moth(TM) to tell the viewer to hide in lockers when the Supermatter Crystal has delaminated. Running away might be a better strategy. It's signed by 'AspEv'."
 	icon_state = "poster44_legit"
 
+/obj/structure/sign/poster/official/disk
+	name = "The Nuke Disk"
+	desc = "This poster is designed for the captain, as a friendly reminder to secure the disk. It's signed by 'ether'."
```
Comment (itsmeow) [inline code/game/objects/effects/contraband.dm:662]: Do not sign the poster, please...

**Review by itsmeow — CHANGES_REQUESTED** [review]

File: code/game/objects/effects/contraband.dm (hunk 1 above)
Comment (Sinestia) [reply code/game/objects/effects/contraband.dm:662]: narcissistic lol

Comment (itsmeow) [reply code/game/objects/effects/contraband.dm:662]: nah, the other ones are signed so it's fair.

**Review by itsmeow — DISMISSED** [review]

**Comment by PowerfulBacon** [comment]
Something feels wrong with the diffs. Might be worth resetting the maps and then recreating them

**Review by PowerfulBacon — CHANGES_REQUESTED** [review]
There should be 1 per map, and the posters floating in space should be removed.

**Comment by itsmeow** [comment]
Cycling because MDB is not working

**Pull request closed without merging**